profiles/base.yaml

```diff
@@ -115,6 +115,7 @@ dump_junk: false
 dump_item_count: 9
 use_barb_combos: false
 whirlwind_trainables:
+offensive_spell_training_max_threshold:
```

Comment:
I think this should default to 32, then line 1304 in combat-trainer could be simplified

Reply:
I just copied the other implementation. I agree but followed suit.